lib/generate_localized.dart

```diff
@@ -72,6 +72,20 @@ class MessageGeneration {
   /// for the [translations] in [locale] and put it in [targetDir].
   void generateIndividualMessageFile(String basicLocale,
       Iterable<TranslatedMessage> translations, String targetDir) {
+    final content = generateIndividualMessageFileContent(
+        basicLocale, translations);
+
+    // To preserve compatibility, we don't use the canonical version of the
+    // locale in the file name.
+    var filename = path.join(
+        targetDir, "${generatedFilePrefix}messages_$basicLocale.dart");
+    new File(filename).writeAsStringSync(content);
+  }
+
+  /// Generate a string that containts the dart code
+  /// with the [tranlsations] in [locale].
+  String generateIndividualMessageFileContent(String basicLocale,
```

Comment:
This name is getting even longer and harder to distinguish from the caller at a glance. Whoever wrote these names originally <grin> could have been terser and clearer. "generate" really doesn't seem right for the caller. What about contentForLocale(...)? I'd rename the caller too, but that's making this change larger than it needs to be.